### PR TITLE
[CF-448] Checking the configuration of copy engines

### PR DESCRIPTION
**Pull request by SVilgelm**
Check that bbcp files exist.
Warn if rsync port option has a default value if direct_trasfer is enabled.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry/lib/os/actions/check_copy_engine_configuration.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            self.check_bbcp()
+        elif copy_backend == 'rsync':
+            self.check_rsync()
+
+    def check_bbcp(self):
+        path = {self.cfg.bbcp.path, self.cfg.bbcp.src_path,
+                self.cfg.bbcp.dst_path}
+        if not all(os.path.isfile(p) for p in path):
+            raise exception.InvalidConfigException(
+                "The path of bbcp are not valid: {path}", path=path)
```
Comment (roman-verchikov) [inline cloudferry/lib/os/actions/check_copy_engine_configuration.py]: Maybe "Invalid path '{path}' specified in `[bbcp] path` config option: {error_message}"?

Comment (SVilgelm) [inline cloudferry/lib/os/actions/check_copy_engine_configuration.py]: Done

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

**Comment by SVilgelm** [comment]
retest

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
grizzly_to_juno job failed because of this [issue](https://github.com/MirantisWorkloadMobility/CloudFerry/pull/1040). overall job passed successfully.

**Comment by SVilgelm** [comment]
retest

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
👍 can be merged

**Pull request merged**